src/battle_util.c

```diff
@@ -7964,7 +7964,9 @@ static u16 CalcMoveBasePower(u16 move, u8 battlerAtk, u8 battlerDef)
         // todo
         break;
     case EFFECT_FLING:
-        // todo: program Fling + Unburden interaction
+    #ifdef ITEM_EXPANSION
```

Comment:
Now that `ItemId_GetFlingPower` is part of the PR, there's no need for the item expansion check